web-app/src/components/Dictonary/Popover.js

```diff
@@ -0,0 +1,42 @@
+import Popover from "@material-ui/core/Popover";
```

Comment:
This can be refactored and made as a common component Popover.js which can be used everywhere, Also bobby has another logic he has used for metadata which I just saw, maybe we can discuss it on Monday.